### PR TITLE
Annotations,labels and enabled flags on post install upgrade jobs

### PR DESCRIPTION
**Pull request by abhilashjoseph**
## Motivation
Add annotations and labels on post install and upgrade jobs, also use enabled flag on the jobs values file to toggle the job creation.

### CODE REVIEW

**Comment by abhilashjoseph** [comment]
Can I please get someone to take a look at this one?

**Review by RafalKorepta — APPROVED** [review]

**Comment by abhilashjoseph** [comment]
Thank you @RafalKorepta, also the workflow is waiting for approval to run, could you please take a look at that as well.

**Pull request merged**
